azure-kusto-ingest/azure/kusto/ingest/ingest_client.py

```diff
@@ -50,6 +55,8 @@ def ingest_from_file(self, file_descriptor: Union[FileDescriptor, str], ingestio
         :param file_descriptor: a FileDescriptor to be ingested.
         :param azure.kusto.ingest.IngestionProperties ingestion_properties: Ingestion properties.
         """
+        super().ingest_from_file(file_descriptor, ingestion_properties)
```

Comment:
Is this a bug fix?
What happened before?
(Same for below stream API)

Reply:
Not a bug fix. Before this commit, the super() method was fully abstract. Now it checks if the ingestor is closed, so we need to go to base.